cmd/server.go

```diff
@@ -147,7 +149,7 @@ func createScheduler() {
 func runServer(cmd *cobra.Command, args []string) error {
 	// Random sleep between 0 to 30 seconds to handle multiple instances starting at the same time
 	sleep := rand.IntN(30)
-	log.Printf("sleeping for %v seconds to handle multiple instances starting at the same time", sleep)
+	log.Info("Sleeping to handle multiple instances starting at the same time", slog.Int("sleep_duration_sec", sleep))
```

Comment:
```suggestion
	log.debug("Sleeping to handle multiple instances starting at the same time", slog.Int("sleep_duration_sec", sleep))
```

Reply:
Also, consider extracting `sleep_duration_sec` to some internal `Logging.Constants` thingy.

Reply:
Created a constants package

Reply:
I actually don't really like this. the point of passing these k/v pairs to the logger is to enhance the information we log in different messages, i don't think it makes sense to create constants out of each of these keys as they are mostly used once and only for logging and make adding additional k/v pairs more complex.

Reply:
If it's used once, then yeah, not need to extract.